dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1010,6 +1011,9 @@ void preValidateLaunchArguments(List<String> args) {
         List<String> argsCopy = new ArrayList<>(args);
         String jsonFile = optVal(argsCopy, "--json", null);
         String yamlFile = optVal(argsCopy, "--yaml", null);
+        conditionalErrorMessage((jsonFile != null) && (yamlFile != null),
+                "Specify a test parameter file with either --json or --yaml not both",
```

Comment:
I'd add a comma after --yaml, i.e., `or --yaml, not both`. 

Also, I realize the message in this context is for when they're trying to specify at least one parameter file, but technically the other option is not to specify a parameter file at all. 

How about:

When specifying a test parameter file, use either --json or --yaml, but not both.

Reply:
"When": seems a little subtle, maybe 
"If specifying a test parameter file, use either --json or --yaml, but not both."?